tests/e2e/common-actions/browser-actions.ts

```diff
@@ -40,4 +39,17 @@ export class BrowserActions {
             ? await t.expect(browserPage.tooltip.textContent).contains(expectedText, `"${expectedText}" Text is incorrect in tooltip`)
             : await t.expect(browserPage.tooltip.textContent).notContains(expectedText, `Tooltip still contains text "${expectedText}"`);
     }
+    /**
+     * Verify that the new key is displayed at the top of the list of keys and opened and pre-selected in List view
+     * */
+    async verifyKeyDisplayedTopAndOpened(keyName: string): Promise<void> {
+        await t.expect(Selector('[aria-rowindex="1"]').withText(keyName).visible).ok(`element with ${keyName} is not visible in the top of list`);
+        await t.expect(Selector('[data-testid="key-name-text"]').withText(keyName).visible).ok(`element with ${keyName} is not opened`);
+    }
+    /**
+     * Verify that the new key is not displayed at the top of the list of keys and opened and pre-selected in List view
+     * */
+    async verifyKeyIsNotDisplayedTop(keyName: string): Promise<void> {
+        await t.expect(Selector('[aria-rowindex="1"]').withText(keyName).visible).notOk(`element with ${keyName} is not visible in the top of list`);
```

Comment:
in testCafe tests with .visible + notOK waits for 5 seconds by default - and it increases test run time
use .exists here instead of .visible